example/docker-sync.yml

```diff
@@ -74,4 +77,4 @@ syncs:
     sync_userid: 'from_host'
     unison-dualside_host_ip: 'auto'
     unison-dualside_host_server_port: 6000
-    sync_strategy: 'unison-dualside'
\ No newline at end of file
+    sync_strategy: 'unison-dualside'
```

Comment:
It's a part of `unison-dualside-sync`, didn't change anything here, my vim removed empty-spaces/newline at the end of file, sorry for that :)